newsletter-april-2021.rst

```diff
@@ -0,0 +1,66 @@
+.. post:: April 5, 2021
+   :tags: newsletter
+   :author: Juan Luis
+   :location: Madrid
+
+.. meta::
+   :description lang=en:
+      xxx
+
+Read the Docs newsletter - April 2021
+=====================================
+
+xxx
+
+Company highlights
+------------------
+
+-  We have a new colleague! Juan Luis will be working with us as
+   Developer Advocate to …
+-  xxx we saw an increment of X % page views
+-  We have `a new status page <http://status.readthedocs.com/>`__ so you
+   can have a glance at how our service is working.
+
+New features
+------------
+
+-  There is a `new support form <https://readthedocs.org/support/>`__ in
+   case you need to send us a request related to your project or
+   account, like asking for more resources, change the project slug, and
+   so forth.
+-  You can now build your project using Python 3.9 by setting your build
+   image to ``testing``. Please `open an
+   issue <https://github.com/readthedocs/readthedocs.org/issues/new>`__
+   if you find any problems with it.
+-  We added the possibility to use the latest version of MkDocs. `Let us
+   know <https://readthedocs.org/support/>`__ if you want to enable it
+   for your project.
+-  We expanded a few areas of our user documentation: we added `a brand
+   new guide on reproducible
+   builds <https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html>`__,
+   considerably improved our `guide on using
+   conda <https://docs.readthedocs.io/en/stable/guides/conda.html>`__,
+   and `started recommending MyST for writing Markdown with
+   Sphinx <https://docs.readthedocs.io/en/stable/intro/getting-started-with-sphinx.html#using-markdown-with-sphinx>`__.
+
+Current focus
+-------------
+
+-  xxx commercial site reliability
+-  xxx security
+
+Upcoming features
```

Comment:
Linking to a specific GitHub issue for each item, if it exists, would be nice to improve community involvement and increase transparency.

Reply:
Added some links when appropriate, will keep adding them as soon as I get some extra pointers. Thanks for the suggestion!